k8s-inception-demo.yml

```diff
@@ -1,15 +1,17 @@
 ---
-- hosts: gateway
+- hosts: localhost
   vars:
-    master_user: "{{ hostvars[groups['kube-master'][1]]['ansible_user'] }}"
-    master_ip: "{{ hostvars[groups['kube-master'][1]]['ip'] }}"
+    node_ip: "{{ hostvars[groups['kube-node'][0]]['ansible_host'] }}"
+    gateway_ip: "{{ hostvars['gateway']['ansible_host'] }}"
   tasks:
   - name: Kill existing forwards
-    shell: "ps aux | grep -v grep | grep 8888:localhost:31000 | awk '{print $2}' | sudo xargs --no-run-if-empty kill -9"
+    shell: "ps aux | grep -v grep | grep 8888:{{ node_ip }}:31000 | awk '{print $2}' | xargs --no-run-if-empty kill -9"
+    ignore_errors: yes
   - name: Forward SSH port
-    shell: "ssh -o StrictHostKeyChecking=no -fNT -L*:8888:localhost:31000 {{ master_user }}@{{ master_ip }}"
+    shell: "ssh -o StrictHostKeyChecking=no -fNT -L*:8888:{{ node_ip }}:31000 {{ gateway_ip }}"
 
-- hosts: kube-master[1]
+- hosts: kube-master[0]
```

Comment:
I don't think we should be executing anything on the k8s master. We shouldn't assume that we have SSH access to it as a user - all interactions should be via the API. This could be from the gateway, or localhost if no gateway is required (e.g. in the Verne environment there is a VPN rather than a gateway).